web/src/components/schema/Object.vue

```diff
@@ -61,8 +87,20 @@ export default defineComponent({
       );
     });
 
+    function renderListOfObjects(value: any[]) {
+      const newVal:any = _.map(value, (v) => v);
+      return newVal;
+    }
```

Comment:
What is the purpose of this function? It looks like it just creates a shallow copy of the input array.

At the very least, this function needs better typing.

Reply:
sorry, this is redundant. I forgot to delete it.